docs/adr/adr-001-predevnet-celestia-node.md

```diff
@@ -0,0 +1,137 @@
+# ADR for Devnet Celestia Node
+
+## Authors
+
+@renaynay @Wondertan @liamsi
+
+## Changelog
+
+* 2021-19-08: initial draft
+
+## Context
+
+This ADR describes a basic pre-devnet design for a "Celestia Node" that was decided at the August 2021 Kyiv offsite that will ideally be completed in early November 2021 and tested in the first devnet.
+
+The goal of this design is to get a basic structure of "Celestia Node" interoperating with a "Celestia Core" consensus node by November 2021 (devnet). 
+
+After basic interoperability on devnet, there will be an effort to merge consensus functionality into the "Celestia Node" design as a modulor service that can be added on top of the basic functions of a "Celestia Node".
+
+## Decision
+
+A "Celestia Node" will be distinctly different than a "Celestia Core" node in the initial implementation for devnet, with plans to merge consensus core functionality into the general design of a "Celestia Node", just added as an additional `ConsensusService` on the node.
```

Comment:
Just a note: for actually "merging" in the full functionality into the Celestia node, we also need a mempool in the Celestia node (which is used by the consensus but not directly part of it).

Reply:
Then they might run a `lightConsensusService` which would just be the tx service portion of the consensus messages. Does it need to be included into this ADR?